### PR TITLE
support other serialization type for bolt

### LINKED ISSUE #504: bolt为什么不支持自己扩展序列化方式？
### Your question

describe your question clearly

### Your scenes

describe your use scenes (why need this feature)

### Your advice

describe the advice or solution you'd like

### Environment

- SOFARPC version:
- JVM version (e.g. `java -version`):
- OS version (e.g. `uname -a`):
- Maven version:
- IDE version:

### PR DESCRIPTION
**Pull request by minli04g**
### Motivation:

support serialization framework for bolt implementation.

### Modification:

BoltClientProxyInvoker.parseSerializeType call super.parseSerializeType

### Result:

Fixes #504

### CODE REVIEW

**Comment by ujjboy** [comment]
Can you add a test case for this change?

**Comment by ujjboy** [comment]
No,  just add a unit test case  in `bootstrap-bolt`,   `test-integration` is for the integration test case,

**Comment by minli04g** [comment]
> No, just add a unit test case in `bootstrap-bolt`, `test-integration` is for the integration test case,

ok

**Comment by minli04g** [comment]
why the codecov/patch is still failing?

**Comment by leizhiyuan** [comment]
it means the coverage of patch is not right , please ignore it this time.

File: extension-impl/bootstrap-bolt/pom.xml [hunk 1]
```diff
@@ -22,6 +22,22 @@
             <artifactId>sofa-rpc-extension-common</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+         <dependency>
+            <groupId>com.alipay.sofa</groupId>
+            <artifactId>sofa-rpc-remoting-bolt</artifactId>
+            <version>${project.parent.version}</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (ujjboy) [inline extension-impl/bootstrap-bolt/pom.xml]: it's unnecessary to add `sofa-rpc-remoting-bolt` dependency.

try to add test resource files, refer https://github.com/alipay/sofa-rpc/tree/master/extension-impl/bootstrap-dubbo/src/test/resources

**Comment by leizhiyuan** [comment]
https://github.com/alipay/sofa-rpc/issues/504

File: extension-impl/bootstrap-bolt/pom.xml (hunk 1 above)
Comment (minli04g) [reply extension-impl/bootstrap-bolt/pom.xml]: done.

**Comment by leizhiyuan** [comment]
Thanks for your contribution

**Pull request merged**